edenia/antelope-tools
# Feat/improve about image 1321

### Improve about image

### What does this PR do?

- Resolve #1321
- Fix colors for dark mode on bp.json generator page

### Steps to test

1. Run the project locally
2. Go to About page
3. Check the new image in dark and light mode
4. Go to bp.json Generator page
5. Check the colors of preview section and drop area

### Screenshots

# About page

## Desktop version
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/e2277162-85f6-499f-ae60-65ed7de95db6)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/6f07404d-ee69-48de-a5bf-ce07f4f58d67)

## Mobile version
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/adb3032f-9216-42c1-814a-8d89272b478e)

# BP JSON Generator page

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/c66b7563-6bb7-4e3b-aaba-3f03596f0cd3)
